### PR TITLE
Typos: `test/SourceKit`

### PR DESCRIPTION
**Pull request by Sajjon**
Fix typos in `test/SourceKit`

This is one batch of many PRs fixing typos, [see the tracking issue.](https://github.com/swiftlang/swift/issues/75022)

### CODE REVIEW

File: test/SourceKit/CodeComplete/complete_filter.swift [hunk 1]
```diff
@@ -52,14 +52,14 @@ func foo() {
 
 // RUN: %sourcekitd-test -req=complete.open -pos=11:5 \
 // RUN:   -req-opts=filtertext=d %s -- %s > %t.d
-// RUN: %FileCheck -check-prefix=CHECKD %s < %t.d
```
Comment (bnbarham) [inline test/SourceKit/CodeComplete/complete_filter.swift:55]: This is `D` as in `%t.d`, no need to rename

File: test/SourceKit/CodeComplete/complete_typerelation.swift.convertible.response [hunk 2]
```diff
@@ -26,40 +26,40 @@
...
-      key.name: "intanceReturnMyEnum(:)",
-      key.description: "intanceReturnMyEnum(self: MyEnum)",
+      key.name: "instanceReturnMyEnum(:)",
+      key.description: "instanceReturnMyEnum(self: MyEnum)",
       key.typename: "() -> MyEnum",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.unrelated,
       key.num_bytes_to_erase: 0,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO013intanceReturncD0ACyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO013instanceReturncD0ACyF",
```
Comment (bnbarham) [inline test/SourceKit/CodeComplete/complete_typerelation.swift.convertible.response:35]:
```suggestion
      key.associated_usrs: "s:21complete_typerelation6MyEnumO014instanceReturncD0ACyF",
```

The number there is the length of the name, so needs +1 for the extra character

File: test/SourceKit/CodeComplete/complete_typerelation.swift.convertible.response [hunk 3]
```diff
@@ -26,40 +26,40 @@
...
-      key.name: "intanceReturnVoid(:)",
-      key.description: "intanceReturnVoid(self: MyEnum)",
+      key.name: "instanceReturnVoid(:)",
+      key.description: "instanceReturnVoid(self: MyEnum)",
       key.typename: "() -> Void",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.invalid,
       key.num_bytes_to_erase: 0,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO17intanceReturnVoidyyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO17instanceReturnVoidyyF",
```
Comment (bnbarham) [inline test/SourceKit/CodeComplete/complete_typerelation.swift.convertible.response:47]:
```suggestion
      key.associated_usrs: "s:21complete_typerelation6MyEnumO18instanceReturnVoidyyF",
```

File: test/SourceKit/CodeComplete/complete_typerelation.swift.convertible.response [hunk 4]
```diff
@@ -26,40 +26,40 @@
...
-      key.description: "intanceReturnVoidDeprecated(self: MyEnum)",
+      key.name: "instanceReturnVoidDeprecated(:)",
+      key.description: "instanceReturnVoidDeprecated(self: MyEnum)",
       key.typename: "() -> Void",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.invalid,
       key.num_bytes_to_erase: 0,
       key.not_recommended: 1,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO27intanceReturnVoidDeprecatedyyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO27instanceReturnVoidDeprecatedyyF",
```
Comment (bnbarham) [inline test/SourceKit/CodeComplete/complete_typerelation.swift.convertible.response:60]:
```suggestion
      key.associated_usrs: "s:21complete_typerelation6MyEnumO28instanceReturnVoidDeprecatedyyF",
```

File: test/SourceKit/CodeComplete/complete_typerelation.swift.identical.response [hunk 5]
```diff
@@ -26,40 +26,40 @@
...
-      key.name: "intanceReturnMyEnum(:)",
-      key.description: "intanceReturnMyEnum(self: MyEnum)",
+      key.name: "instanceReturnMyEnum(:)",
+      key.description: "instanceReturnMyEnum(self: MyEnum)",
       key.typename: "() -> MyEnum",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.unrelated,
       key.num_bytes_to_erase: 0,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO013intanceReturncD0ACyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO013instanceReturncD0ACyF",
```
Comment (bnbarham) [inline test/SourceKit/CodeComplete/complete_typerelation.swift.identical.response:35]:
```suggestion
      key.associated_usrs: "s:21complete_typerelation6MyEnumO014instanceReturncD0ACyF",
```

File: test/SourceKit/CodeComplete/complete_typerelation.swift.identical.response [hunk 6]
```diff
@@ -26,40 +26,40 @@
...
-      key.name: "intanceReturnVoid(:)",
-      key.description: "intanceReturnVoid(self: MyEnum)",
+      key.name: "instanceReturnVoid(:)",
+      key.description: "instanceReturnVoid(self: MyEnum)",
       key.typename: "() -> Void",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.invalid,
       key.num_bytes_to_erase: 0,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO17intanceReturnVoidyyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO17instanceReturnVoidyyF",
```
Comment (bnbarham) [inline test/SourceKit/CodeComplete/complete_typerelation.swift.identical.response:47]:
```suggestion
      key.associated_usrs: "s:21complete_typerelation6MyEnumO18instanceReturnVoidyyF",
```

File: test/SourceKit/CodeComplete/complete_typerelation.swift.identical.response [hunk 7]
```diff
@@ -26,40 +26,40 @@
...
-      key.description: "intanceReturnVoidDeprecated(self: MyEnum)",
+      key.name: "instanceReturnVoidDeprecated(:)",
+      key.description: "instanceReturnVoidDeprecated(self: MyEnum)",
       key.typename: "() -> Void",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.invalid,
       key.num_bytes_to_erase: 0,
       key.not_recommended: 1,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO27intanceReturnVoidDeprecatedyyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO27instanceReturnVoidDeprecatedyyF",
```
Comment (bnbarham) [inline test/SourceKit/CodeComplete/complete_typerelation.swift.identical.response:60]:
```suggestion
      key.associated_usrs: "s:21complete_typerelation6MyEnumO28instanceReturnVoidDeprecatedyyF",
```

File: test/SourceKit/CursorInfo/cursor_generated_interface.swift [hunk 8]
```diff
@@ -105,13 +105,13 @@ public class CType {}
 @interface DType
 @end
 
-// RUN: %sourcekitd-test -req=interface-gen-open -module LibA -- -F %t/frameworks -target %target-triple == -req=cursor -pos=14:57 -print-raw-response | %FileCheck %s --check-prefix=CHECKD
```
Comment (bnbarham) [inline test/SourceKit/CursorInfo/cursor_generated_interface.swift:108]: This is `D` as in the `D` module, no need to rename.

**Review by bnbarham — COMMENTED** [review]
Thanks @Sajjon!

File: test/SourceKit/CursorInfo/cursor_generated_interface.swift (hunk 8 above)
Comment (Sajjon) [reply test/SourceKit/CursorInfo/cursor_generated_interface.swift]: Ah, sorry, thanks! I've reverted. (I think that is what you meant, right?)

File: test/SourceKit/CodeComplete/complete_filter.swift (hunk 1 above)
Comment (Sajjon) [reply test/SourceKit/CodeComplete/complete_filter.swift]: Ah, sorry, thanks! I've reverted. (I think that is what you meant, right?)

Comment (bnbarham) [reply test/SourceKit/CodeComplete/complete_filter.swift]: Yep! Thanks.

**Comment by bnbarham** [comment]
@swift-ci please test

**Review by bnbarham — APPROVED** [review]

**Comment by ahoppen** [comment]
@swift-ci Please smoke test macOS

File: test/SourceKit/InterfaceGen/error_swift_module.swift [hunk 9]
```diff
@@ -45,14 +45,14 @@ import NonExistant
...
-// RUN: not %sourcekitd-test -req=interface-gen -module NonExistant -- -I %t -target %target-triple %s 2>&1 | %FileCheck --check-prefix DIRECT-NONEXISTANT %s
-// DIRECT-NONEXISTANT: Could not load module: NonExistant
+// RUN: not %sourcekitd-test -req=interface-gen -module NonExistent -- -I %t -target %target-triple %s 2>&1 | %FileCheck --check-prefix DIRECT-NONEXISTENT %s
+// DIRECT-NONEXISTENT: Could not load module: NonExistent
 
-// RUN: %target-swift-frontend -emit-module %t/Inputs/Transitive.swift -module-name NonExistant -o %t
-// RUN: %target-swift-frontend -emit-module %t/Inputs/LibraryNonExistant.swift -module-name LibraryNonExistant -I %t -o %t
+// RUN: %target-swift-frontend -emit-module %t/Inputs/Transitive.swift -module-name NonExistent -o %t
+// RUN: %target-swift-frontend -emit-module %t/Inputs/LibraryNonExistant.swift -module-name LibraryNonExistent -I %t -o %t
 // RUN: rm -rf %t/NonExistant.swiftmodule
```
Comment (ahoppen) [inline test/SourceKit/InterfaceGen/error_swift_module.swift:55]: You missed two renames here. While debugging the failure, I also noticed that we were carrying state between executions of this test. Could you fix that as well by adding `// RUN: %empty-directory(%t)` as the first line of this test file (before `// RUN: %empty-directory(%t/Inputs)`).

```suggestion
// RUN: %target-swift-frontend -emit-module %t/Inputs/LibraryNonExistent.swift -module-name LibraryNonExistent -I %t -o %t
// RUN: rm -rf %t/NonExistent.swiftmodule
```

**Comment by ahoppen** [comment]
@swift-ci Please smoke test

**Review by ahoppen — APPROVED** [review]
Thanks for fixing the typos, @Sajjon 🙏🏽

**Pull request merged**